### PR TITLE
added safe option for selecting gas price

### PR DESCRIPTION
**Pull request by rango527**

### CODE REVIEW

File: packages/client/src/services/api-etherscan.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
```
Comment (luckyOdisetti) [inline packages/client/src/services/api-etherscan.ts:1]: we've been using native `fetch` in the codebase

Comment (rango527) [reply packages/client/src/services/api-etherscan.ts:1]: Let me remove axios and use native fetch.

**Comment by luckyOdisetti** [comment]
@rango527 can we get a description on the PR of what and why. We currently use `gasnow` ws feed. iirc we were on ethgasstation and were having issues with it before moving forward.

**Comment by rango527** [comment]
> @rango527 can we get a description on the PR of what and why. We currently use `gasnow` ws feed. iirc we were on ethgasstation and were having issues with it before moving forward.

Hi @luckyOdisetti , sometimes we used to get following errors while estimating gas.
"Could not estimate gas: err: max fee per gas less than block base fee."
https://sentry.io/organizations/peggyjv-inc/issues/2539738352/?environment=production&project=5618374&query=is%3Aunresolved
And such an issue was also occurred at DancingBananas project also.
And we applied higher gas price by fetching gas info from https://ethgasstation.info/json/ethgasAPI.json.
I just tried to apply it to sommelier project and there were already 3 options for gas selection.
So I just added one option more for safe gas selection.
But I think we should discuss more in detail about it.

File: packages/client/src/services/api-etherscan.ts (hunk 1 above)
Comment (taariq) [reply packages/client/src/services/api-etherscan.ts:1]: OOh. that's cool!

**Comment by rango527** [comment]
Unfortunately looks like it will not work well. And I think this 'safe' option is useless at this point.
I am abandoning this PR.

**Pull request closed without merging**